proj/src/reducers/statistics/statisticsReducer.js

```diff
@@ -1,5 +1,11 @@
 import { handleActions, combineActions } from 'redux-actions';
+import { set } from 'lodash/fp'
+const initialState = {
+  data: []
```

Comment:
too much 'data' here. like I said, not the best name. try to choose a different data structure here. what if you just made this entire reducer return an array